consensus/bor/api.go

```diff
@@ -105,3 +111,41 @@ func (api *API) GetCurrentValidators() ([]*Validator, error) {
 	}
 	return snap.ValidatorSet.Validators, nil
 }
+
+// GetRootHash gets the current validators
+func (api *API) GetRootHash(start uint64, end uint64) ([]byte, error) {
```

Comment:
It should also check the latest block height as part of validation.

Reply:
We should also limit `end - start + 1` to avoid abuse. Example: `end - start + 1 < 16384`

Reply:
What limit do we want to keep? This will prevent being able to send large checkpoints.

Reply:
@atvanguard Yeah. That's why we want to make it big but not too big where one can call these APIs with huge checkpoint and cause the node stoppage or something.

We can either limit APIs call time or something.